app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+const DeleteAlert = ({ onClose }) => {
+  const handleDelete = () => {
+    Toastr.success("Contact Deleted Successfully");
+    onClose();
+  };
+
+  return (
+    <Alert
+      isOpen
+      message="Are you sure you want to continue? This cannot be undone."
```

Comment:
I am hoping that you will add localizations later. But do not forget to cover all the places where translations have to be applied when you keep it for the end.

Reply:
Yes, I am currently working on it currently.